### DISCUSSION TITLE
iOS上AltStore安裝失敗

### DISCUSSION THREAD

**Discussion by Annabel0530** [xiaoyaocz/flutter_dmzj#89] [Q&A]
The name “动漫之家X” contains invalid characters.

**Comment by xiaoyaocz** [comment]
可以用Sideloadly试试

**Comment by oubeichen** [comment]
估计是altstore不支持中文的问题

**Reply by Annabel0530** [reply]
> 估计是altstore不支持中文的问题

刚刚查到了，但是我只有Windows，也不太懂如何解决
https://github.com/altstoreio/AltStore/issues/1150#issuecomment-1453634737

**Reply by oubeichen** [reply]
> > 估计是altstore不支持中文的问题
> 
> 刚刚查到了，但是我只有Windows，也不太懂如何解决 [altstoreio/AltStore#1150 (comment)](https://github.com/altstoreio/AltStore/issues/1150#issuecomment-1453634737)

你不是发了么，windows就是解压修改文件然后重新打包再altstore安装就行了。虽然我一直不知道altstore怎么从电脑端方便安装，都丢到icloud里面再用设备打开

**Reply by Annabel0530** [reply]
> > > 估计是altstore不支持中文的问题
> > 
> > 
> > 刚刚查到了，但是我只有Windows，也不太懂如何解决 [altstoreio/AltStore#1150 (comment)](https://github.com/altstoreio/AltStore/issues/1150#issuecomment-1453634737)
> 
> 你不是发了么，windows就是解压修改文件然后重新打包再altstore安装就行了。虽然我一直不知道altstore怎么从电脑端方便安装，都丢到icloud里面再用设备打开

我win用的bandizip....执行解压缩再打包改名的ipa没办法安装，不过已经拖人帮我处理好啦
